### PR TITLE
[DOC-133] Updated node disk usage and added other new alerts

### PR DESCRIPTION
**Pull request by aishwarya24**
Build preview : https://deploy-preview-19821--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates

Summary:

1. Added the following two alerts instead of DB node disk usage:

     - DB node data disk usage

     - DB node system disk usage

2. Added other new alerts. 

To do - Update existing ones (another follow-up PR)

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 1]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
 max by (node_prefix) (count by (node_prefix, exported_instance) (max_over_time(yb_node_leaderless_tablet{node_prefix="$node_prefix"}[5m])) > 0)
 ```
 
+### Backup Deletion failure
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:305]:
```suggestion
### Backup deletion failure
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 2]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+Failed to delete `$value` backups for customer `'yugabyte support'` in last GC run. Check logs for more details.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:307]: Customer yugabyte support?

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 3]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+### Number of YEDIS connections is high
+
+Number of YEDIS connections for universe `'$universe_name'` is above `$threshold`. Current value is `$value`.
+
+```expression
+max by (universe_uuid) (max_over_time(rpc_connections_alive{universe_uuid="__universeUuid__",export_type="cql_export"}[5m])) {{ query_condition }} {{ query_threshold }}
+```
+
+### Memory Consumption
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:362]:
```suggestion
### Memory consumption
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 4]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+### Under-replicated master
+
+Master is missing from raft group or has follower lag higher than `$threshold` seconds for universe `'$universe_name'`.
+
+```expression
+(min_over_time((ybp_universe_replication_factor{universe_uuid='{{ $labels.universe_uuid }}'} - on(universe_uuid) count by(universe_uuid) (count by (universe_uuid, exported_instance) (follower_lag_ms{export_type="master_export", universe_uuid='{{ $labels.universe_uuid }}'})))[{{query_threshold }}s:]) > 0 or (max by(universe_uuid) (follower_lag_ms{export_type="master_export", universe_uuid='{{ $labels.universe_uuid }}'}) {{ query_condition }} ({{ query_threshold }} * 1000)))
+```
+
+### New YSQL Tables Added
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:384]:
```suggestion
### New YSQL tables added
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 5]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+      - ignoring (saved_name) (avg_over_time(node_memory_Cached_bytes{universe_uuid="__universeUuid__"}[10m]))
+      - ignoring (saved_name) (avg_over_time(node_memory_MemFree_bytes{universe_uuid="__universeUuid__"}[10m]))
+      - ignoring (saved_name) (avg_over_time(node_memory_Slab_bytes{universe_uuid="__universeUuid__"}[10m])))
+      / ignoring (saved_name) (avg_over_time(node_memory_MemTotal_bytes{universe_uuid="__universeUuid__"}[10m])))
+      * 100 {{ query_condition }} {{ query_threshold }}
+```
+
+### Under-replicated master
+
+Master is missing from raft group or has follower lag higher than `$threshold` seconds for universe `'$universe_name'`.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:378]:
```suggestion
Master is missing from RAFT group or has follower lag higher than `$threshold` seconds for universe `'$universe_name'`.
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 6]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+### New YSQL Tables Added
+
+New YSQL tables are added to the source universe `'$universe_name'` in the database with an existing xCluster configuration, but not added to the xCluster replication.
+
+```expression
+((count by (namespace_name, universe_uuid)(count by(namespace_name, table_id, universe_uuid)(rocksdb_current_version_sst_files_size{universe_uuid="__universeUuid__",table_type="PGSQL_TABLE_TYPE"}))) - count by(namespace_name, universe_uuid)(count by(namespace_name, universe_uuid, table_id)(async_replication_sent_lag_micros{universe_uuid="__universeUuid__",table_type="PGSQL_TABLE_TYPE"}))) {{ query_condition }} {{ query_threshold }}
+```
+
+### PITR Config Failure
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:392]:
```suggestion
### PITR config failure
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 7]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+Last Snapshot task for universe `'$universe_name'` failed. To retry, check PITR Configuration task result for more details.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:394]:
```suggestion
Last snapshot task for universe `'$universe_name'` failed. To retry, check PITR configuration task result for more details.
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 8]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+```expression
+min(ybp_pitr_config_status{universe_uuid = "__universeUuid__"}) {{ query_condition }} 1
+```
+
+### Private access key permission status
+
+Invalid Permissions of private access key file for universe `'$universe_name'`. You need to check YugabyteDB Anywhere logs for details or contact {{% support-platform %}}.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:402]:
```suggestion
Invalid permissions of private access key file for universe `'$universe_name'`. You need to check YugabyteDB Anywhere logs for details or contact {{% support-platform %}}.
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 9]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+```expression
+last_over_time(ybp_universe_private_access_key_status{universe_uuid = "__universeUuid__"}[1d]) {{ query_condition }} 1
+```
+
+### Replication Lag
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:408]:
```suggestion
### Replication lag
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 10]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+Average replication lag for universe `'$universe_name'` is above `$threshold` milliseconds. Current value is `$value` milliseconds.
+
+```expression
+max by (universe_uuid) (avg_over_time(async_replication_committed_lag_micros{universe_uuid="__universeUuid__"}[10m]) or avg_over_time(async_replication_sent_lag_micros{universe_uuid="__universeUuid__"}[10m])) / 1000 {{ query_condition }} {{ query_threshold }}
+```
+
+### SSH Key Rotation Failure
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:416]:
```suggestion
### SSH key rotation failure
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 11]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+Last SSH Key Rotation task for universe `'$universe_name'` failed. To retry, check SSH Key Rotation task result.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:418]:
```suggestion
Last SSH key rotation task for universe `'$universe_name'` failed. To retry, check SSH key rotation task result.
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 12]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+```expression
+last_over_time(ybp_ssh_key_rotation_status{universe_uuid = "__universeUuid__"}[1d]) {{ query_condition }} 1
+```
+
+### SSH Key expiry
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:424]:
```suggestion
### SSH key expiry
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 13]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+SSH Key for universe `'$universe_name'` will expire in `$value` days.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:426]:
```suggestion
SSH key for universe `'$universe_name'` will expire in `$value` days.
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 14]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+```expression
+ybp_universe_ssh_key_expiry_day{universe_uuid="__universeUuid__"} {{ query_condition }} {{ query_threshold }}
+```
+
+### Metric Collection Failure
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:432]:
```suggestion
### Metric collection failure
```

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 15]
```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
...
+
+Failed to collect metric for universe `'$universe_name'`. You need to check YugabyteDB Anywhere logs for details or contact {{% support-platform %}}.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:434]: just 1 metric? all metrics?

Comment (aishwarya24) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:434]: I think the yml file has a typo. The summary has metric in it. Ill change it to metrics.

**Review by ddhodge — APPROVED** [review]
LGTM

**Pull request merged**